test/OpenTelemetry.Instrumentation.StackExchangeRedis.Tests/StackExchangeRedisCallsInstrumentationTests.cs

```diff
@@ -54,8 +54,10 @@ public void SuccessfulCommandTest(string value)
             using var connection = ConnectionMultiplexer.Connect(connectionOptions);
 
             var activityProcessor = new Mock<BaseProcessor<Activity>>();
+            var sampler = new TestSampler();
```

Comment:
also I noticed this class was not well documented but is pretty useful across our tests, is it worthwhile to add comments?

Reply:
We generally don't document internal test helpers unless they are very special/complex. TestSampler (and many others like TestProcessor..) are not very complex warranting a document on it.